extensions/civetweb/processors/ListenHTTP.cpp

```diff
@@ -191,7 +206,11 @@ void ListenHTTP::onSchedule(core::ProcessContext *context, core::ProcessSessionF
   }
 
   server_.reset(new CivetServer(options, &callbacks_, &logger_));
-  handler_.reset(new Handler(basePath, context, sessionFactory, std::move(authDNPattern), std::move(headersAsAttributesPattern)));
+
+  context->getProperty(BatchSize.getName(), batch_size_);
+  logger_->log_debug("ListenHTTP using %s: %d", BatchSize.getName(), batch_size_);
```

Comment:
this should be `%zu`, too

Reply:
Fixed in [7813ee3](https://github.com/apache/nifi-minifi-cpp/pull/921/commits/7813ee3aa9fc16db0b132d49f27d88bf4671df3f)